presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -163,22 +163,17 @@ groupBy
     ;
 
 groupingElement
```

Comment:
It looks like it should belong to different commit.  One thing is to move the grouping sets expensasion from analyzer to planner and other is to change how grouping sets expressions are parsed, validated and analyzed (before expansions). What do you think?